The-Puffin-Room/src/components/CommentSection.jsx

```diff
@@ -33,6 +33,12 @@ function CommentSection({ article_id }) {
     });
   };
 
+  const onDelete = (commentId) => {
```

Comment:
does this function need to be used elsewhere? if not it makes sense to move this into the comment card and pass set state down to the comment card